frontend/packages/schema-editor/src/components/SchemaInspector/restrictions/NumberRestrictionsReducer.tsx

```diff
@@ -55,15 +55,17 @@ export const validateMinMax = (
   const minEqualMax = Number(formatState.min) === Number(formatState.max);
   if (minMaxAreInclusive && minEqualMax) {
     return NumberRestrictionsError.NoError;
```

Comment:
Jeg tror ikke denne delen er nødvendig. En bedre fremgangsmåte er å sjekke for feilsituasjonene først, og returnere `NoError` til slutt hvis ingen av dem har inntruffet. Da får vi mer lesbar kode. Her returnerer du `NoError` to steder, og det blir fort uoversiktlig.